scripts/configs/pg3_offline_config.yaml

```diff
@@ -0,0 +1,30 @@
+# Offline PG3 Configuration File 
+---
+APPROACHES:
+  pg3:
+    NAME: "pg3"
+    FLAGS: {}
+  pg3-given-ops:
+    NAME: "pg3"
+    FLAGS: 
+      strips_learner: oracle 
+  plan-only:
+    NAME: "oracle"
+    FLAGS: {}
+    
+ENVS:
+  delivery:
+    NAME: "pddl_easy_delivery_procedural_tasks"
+    FLAGS: {}
+  spanner:
+    NAME: "pddl_spanner_procedural_tasks"
+    FLAGS: {}
+
+ARGS: []
+FLAGS:
+  strips_learner: oracle
```

Comment:
is this shared? won't that defeat the purpose of having it on only some of the approaches above? @tomsilver

Reply:
crap yeah thanks good catch!